src/include/pbs_ifl.h

```diff
@@ -581,84 +581,98 @@ DECLDIR char *__pbs_server_location(void);
+DECLDIR struct batch_status *(*pfn_pbs_stathook)(int, char *, struct attrl *, char *);
+DECLDIR struct ecl_attribute_errors * (*pfn_pbs_get_attributes_in_error)(int);
+DECLDIR char *(*pfn_pbs_submit)(int, struct attropl *, char *, char *, char *);
+DECLDIR char *(*pfn_pbs_submit_resv)(int, struct attropl *, char *);
+DECLDIR int (*pfn_pbs_delresv)(int, char *, char *);
+DECLDIR int (*pfn_pbs_terminate)(int, int, char *);
+DECLDIR preempt_job_info *(*pfn_pbs_preempt_jobs)(int, char**);
 
-DECLDIR int pbs_alterjob(int, char *, struct attrl *, char *);
-
-DECLDIR int pbs_connect(char *);
-
-DECLDIR int pbs_connect_extend(char *, char *);
-
-DECLDIR char *pbs_default(void);
-
-DECLDIR int pbs_deljob(int, char *, char *);
-
-DECLDIR int pbs_disconnect(int);
-
-DECLDIR char *pbs_geterrmsg(int);
-
-DECLDIR int pbs_holdjob(int, char *, char *, char *);
-
-DECLDIR char *pbs_locjob(int, char *, char *);
-
-DECLDIR int pbs_manager(int, int, int, char *, struct attropl *, char *);
-
-DECLDIR int pbs_movejob(int, char *, char *, char *);
-
-DECLDIR int pbs_msgjob(int, char *, int, char *, char *);
 
 DECLDIR int pbs_relnodesjob(int, char *, char *, char *);
-
-DECLDIR int pbs_orderjob(int, char *, char *, char *);
-
-DECLDIR int pbs_rerunjob(int, char *, char *);
-
-DECLDIR int pbs_rlsjob(int, char *, char *, char *);
-
-DECLDIR int pbs_runjob(int, char *, char *, char *);
-
-DECLDIR char **pbs_selectjob(int, struct attropl *, char *);
-
-DECLDIR int pbs_sigjob(int, char *, char *, char *);
-
-DECLDIR void pbs_statfree(struct batch_status *);
-
-DECLDIR struct batch_status *pbs_statrsc(int, char *, struct attrl *, char *);
-
-DECLDIR struct batch_status *pbs_statjob(int, char *, struct attrl *, char *);
-
-DECLDIR struct batch_status *pbs_selstat(int, struct attropl *, struct attrl *, char *);
-
-DECLDIR struct batch_status *pbs_statque(int, char *, struct attrl *, char *);
-
-DECLDIR struct batch_status *pbs_statserver(int, struct attrl *, char *);
-
-DECLDIR struct batch_status *pbs_statsched(int, struct attrl *, char *);
-
-DECLDIR struct batch_status *pbs_stathost(int, char *, struct attrl *, char *);
-
-DECLDIR struct batch_status *pbs_statnode(int, char *, struct attrl *, char *);
-
-DECLDIR struct batch_status *pbs_statvnode(int, char *, struct attrl *, char *);
-
-DECLDIR struct batch_status *pbs_statresv(int, char *, struct attrl *, char *);
-
-DECLDIR struct batch_status *pbs_stathook(int , char *, struct attrl *, char *);
-
-DECLDIR struct ecl_attribute_errors * pbs_get_attributes_in_error(int);
-
-DECLDIR char *pbs_submit(int, struct attropl *, char *, char *, char *);
-
-DECLDIR char *pbs_submit_resv(int, struct attropl *, char *);
-
-DECLDIR int pbs_delresv(int, char *, char *);
-
-DECLDIR int pbs_terminate(int, int, char *);
-
 DECLDIR char *pbs_modify_resv(int, char*, struct attropl *, char *);
+<<<<<<< HEAD
 
 DECLDIR preempt_job_info *pbs_preempt_jobs(int, char **);
+=======
+>>>>>>> Refactoring: Making IFL library pluggable
```

Comment:
This conflict still needs to be resolved. I think you can simply remove it since it is declared below.